src/ape/managers/chain.py

```diff
@@ -370,8 +370,12 @@ def pending_timestamp(self) -> int:
         return self.provider.get_block("pending").timestamp
 
     @pending_timestamp.setter
-    def pending_timestamp(self, new_value: int):
-        self.provider.set_timestamp(new_value)
+    def pending_timestamp(self, new_value: str):
+        """
+        Converts date string to timestamp
+        """
+        from ape import convert
```

Comment:
inject `ConversionManager` into `ChainManager`

Reply:
or, leave a `TODO` to do this. @evanblank3 is working on this anyways

Reply:
I'll put a TODO, but is that what we did in AccountHistory? Line 200 of chain.py?

Reply:
oh, yeah. put a TODO there too

Reply:
done